tests/operator_test.go

```diff
@@ -737,6 +738,29 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 				vmis = append(vmis, vmi)
 			}
 
+			lastVMIIndex := len(vmis) - 1
+			vmi := vmis[lastVMIIndex]
+			const nadName = "secondarynet"
+
+			Expect(libnet.CreateNAD(virtClient, vmi.GetNamespace(), nadName)).To(Succeed())
```

Comment:
Can you confirm someone cleans it up on teardown? (I remember there is NAD deletion, but I do not remember if it is per per-defined name or not).

Reply:
The global test suite cleanup delete net-attach-def as well https://github.com/kubevirt/kubevirt/blob/main/tests/testsuite/namespace.go#L190